package.json

```diff
@@ -17,7 +17,8 @@
     "site": "rimraf tmp/site && webpack --env.site --env.element=blaze-elements",
     "commit": "git-cz",
     "commitmsg": "validate-commit-msg",
-    "precommit": "yarn ts:lint && yarn test",
+    "precommit": "npm run ts:lint",
```

Comment:
Also we wanted to move lint to Travis build didn't we? :-)
- I am not sure, whether we have it there already...